.github/workflows/static-checks.yaml

```diff
@@ -79,6 +79,10 @@ jobs:
         echo "Set environment variables for the libseccomp crate to link the libseccomp library statically"
         echo "LIBSECCOMP_LINK_TYPE=static" >> $GITHUB_ENV
         echo "LIBSECCOMP_LIB_PATH=${libseccomp_install_dir}/lib" >> $GITHUB_ENV
+    - name: Install dependency
+      run: |
+        sudo apt update
+        sudo apt install -y libdevmapper-dev
```

Comment:
This should go as a PR to github.com/kata-containers/tests and then you can include this for testing with this repo by adding an entry in your commit: https://github.com/kata-containers/tests#breaking-compatibility.

Reply:
Umm, I don't understand. All the static checks have passed, though?

If this should go to the tests repo, how does it benefit the static checks? The GHA for static checks shows that there is no dependency on anything outside for those checks.

Reply:
Done!